src/test/java/com/wikia/webdriver/testcases/facebooktests/FacebookButtonTests.java

```diff
@@ -0,0 +1,79 @@
+package com.wikia.webdriver.testcases.facebooktests;
+
+import org.testng.annotations.Test;
+
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.dropdowncomponentobject.DropDownComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.signup.SignUpPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialNewFilesPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.login.SpecialUserLoginPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.preferences.PreferencesPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.preferences.PreferencesPageObject.tabNames;
+
+/* 
+ * Check for facebook button on the page
+ */
+
+public class FacebookButtonTests extends NewTestTemplate {
+
+	Credentials credentials = config.getCredentials();
+
+	// check for FB button on login drop down
```

Comment:
Are comments required here?
From my point of view the method name tells us about purposes of the test.